resources/views/footer.blade.php

```diff
@@ -1,24 +1,100 @@
 @props([
-    'wrapperClass'  => 'px-8 max-w-7xl flex md:px-10',
-    'copyClass'     => '',
-    'border'        => false,
-    'copyText'      => null,
-    'isArkProduct'  => true,
-    'socials'       => null,
-    'copyrightSlot' => null,
+    'wrapperClass'  => 'px-8 max-w-7xl md:px-10',
+    'backgroundColor' => 'bg-theme-secondary-900',
+    'linkClass' => 'underline hover:no-underline transition-default hover:text-theme-secondary-200',
+    'name' => null,
+    'url' => null,
```

Comment:
```suggestion
    'copyrightName' => null,
    'copyrightUrl' => null,
```

maybe? since they are only used there from what I saw